tests/fast/TenantLock.toml

```
@@ -0,0 +1,22 @@
+[configuration]
+allowDefaultTenant = false
+tenantModes = ['optional', 'required']
+
+[[test]]
+testTitle = 'TenantCreation'
```

I think CI is failing because of a change in the behavior of clearing after test. It now deletes tenants, so you'll need to prevent that:

```suggestion
testTitle = 'TenantCreation'
clearAfterTest = false
```